Day3Part2.js

```diff
@@ -0,0 +1,53 @@
+const readline = require("readline").createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+readline.question(`enter your data `, function LifeSupportRating(data) {
+  var Data = data.split(" ");
+  zeroCounter = 0;
+  oneCounter = 0;
+
+  let arro2 = [...Data];
+  let arrco2 = [...Data];
+
+  let i = 0;
+  
+  const getMostCommun = (array, i) => {
+    for (let j = 0; j < array.length; j++) {
+      if (Data[j][i] === "0") {
+        zeroCounter += 1;
+      } else {
+        oneCounter += 1;
+      }
+    }
+    if (oneCounter > zeroCounter || oneCounter == zeroCounter) {
+      return 1;
+    } else {
+      return 0;
+    }
+    zeroCounter = 0;
+    oneCounter = 0;
+  };
+
+  function Rating(arr, key) {
+    let i = 0;
+    while (arr.length > 1) {

```

Comment:
Above you used a for loop for the same thing. You should stick to one of the two if you do exactly this type of loop.

```suggestion
    while (arr.length > 1) {
```